Adds the Acera but Correctly This Time

kyres1 opened the pull request:
## About the PR
Title. This is a re-opening of https://github.com/Monolith-Station/Monolith/pull/2424

exact same PR, but with  fixes from post-merge (and revert) feedback by @Avalon-Proto and Articular in mono discord

Those fixes include:
catwalks under thrusters
Fixed the SMES connections
fixed thruster orientation
Gunnery server added

and other fixes include:
Airlocks are labelled
Outgoing scrubber pump exists now
gravgen is properly connected to powernet
Thrusters downsized to fit power output (they are now 4 normal thrusters instead of 2 large thrusters)

## Why / Balance
I was asked to make another merc ship, and also the current ones are pretty one-tone, ugly or tedious to use. I also wanted to add something that made sense to bounty hunt in

## How to test
load it

## Media
See prior pr


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [ X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ X ] I have added media to this PR or it does not require an ingame showcase.
- [ X ] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
hope not

**Changelog**
:cl:
- add: Added the Acera bounty hunting ship but fixed this time

kyres1 committed:
aceraaaaa

kyres1 committed:
adds gravgen and gyroscope and fixes the power

kyres1 committed:
appends desc with an imporatnt detail

monolith8319 committed:
chore: automatically update REUSE headers

kyres1 committed:
adjusts price to 83500

kyres1 committed:
Merge remote-tracking branch 'kyres1-origin/10-16-2025-ship_addition_acera' into 10-16-2025-ship_addition_acera

kyres1 committed:
Many fixes

monolith8319 committed:
chore: automatically update REUSE headers

kasature90 commented on Resources/Prototypes/_Mono/Shipyard/Expedition/acera.yml:
```diff
@@ -0,0 +1,46 @@
+- type: vessel
+  id: Acera
+  parent: BaseVessel
+  name: SKR Acera
```
use a different company prefix

kasature90 reviewed (changes requested):
these rear thrusters need lattice in front of them
<img width="208" height="155" alt="image" src="https://github.com/user-attachments/assets/13b6a60d-0743-463a-9e2d-1a336d2dad24" />

kitchen really needs more decal and/or tile variety
<img width="406" height="581" alt="image" src="https://github.com/user-attachments/assets/8c95a88d-08c9-4cff-836f-e7d6a3e49f68" />

dont map criminal records computer, wardrobe locker needs to start closed, room needs decaling
<img width="408" height="265" alt="image" src="https://github.com/user-attachments/assets/55d1a946-840d-4920-9269-eb2847fec330" />

this stool is facing the wrong way
<img width="333" height="289" alt="image" src="https://github.com/user-attachments/assets/c4d16534-4048-4ae5-be6f-cf8eaaa63e90" />

delivery decals on the floor are kinda random imo, the directional windows on the GCS should probably be removed, + the diagonal decal is Kinda Weird on the mono tile
<img width="265" height="268" alt="image" src="https://github.com/user-attachments/assets/72b81888-2652-44d2-855a-24150cd7132d" />

random clothing pieces (especially on the floor) are weird, tile variety is pretty weak, maybe add a drain
<img width="323" height="247" alt="image" src="https://github.com/user-attachments/assets/29f82431-9fc3-4a05-86c7-18478fc655fb" />

the suit storages & techfab here seem randomly placed, you could replace this w/ catwalks on plating connecting the doors & just mono tiles w/ decals for the suit storage/techfab, also the switches in the bottom next to the cargo bay door are going off the wall
<img width="602" height="176" alt="image" src="https://github.com/user-attachments/assets/9c865a0d-22ae-41a8-859b-11580e779671" />

roundstart power deficit (just increase the targetpower in fuelgeneratorcomponent)
<img width="644" height="336" alt="image" src="https://github.com/user-attachments/assets/5262051e-2b03-41ff-b07f-066370d9c546" />

firelocks on this are redundant + button only opens 1 shutter + button is awkward to press under the sign
<img width="218" height="81" alt="image" src="https://github.com/user-attachments/assets/195081a4-f9a4-46e9-9e1d-3c58a2b252f4" />

3 chairs is probably too much here, gunnery console and exped console should switch position, floor/decal variety is pretty bad, missing holopad
<img width="378" height="270" alt="image" src="https://github.com/user-attachments/assets/231c09f2-2e6d-4622-83c0-782a538edfb8" />

its Obnoxiously Bright here + air can needs to be anchored
<img width="240" height="155" alt="image" src="https://github.com/user-attachments/assets/865291b5-4125-4ae7-9924-8aa2e9a188b2" />

also, every firelock under exterior airlocks w/ fans will Do Nothing and is not needed

kyres1 committed:
starches requested changes

kyres1 committed:
Merge remote-tracking branch 'kyres1-origin/10-16-2025-ship_addition_acera' into 10-16-2025-ship_addition_acera

kyres1 commented:
@kasature90 
Fixed goofy thrusters
<img width="623" height="659" alt="NVIDIA_Overlay_csginA0JHb" src="https://github.com/user-attachments/assets/e8e0f8e9-badd-497a-be5b-8218bf6e54e3" />

Halved the kitchen with wood/kitchen tiles, added a spline to it
<img width="638" height="848" alt="NVIDIA_Overlay_mgJuOdtoK0" src="https://github.com/user-attachments/assets/a92d2085-1ece-476f-83cc-5ffd55380ef3" />

no more criminal records or open closet
<img width="290" height="364" alt="NVIDIA_Overlay_aSrtTAwcOn" src="https://github.com/user-attachments/assets/dccd97d1-05e8-4916-9ad6-a02728a69b52" />

Stool direction adjusted so that the seated person can properly monitor the dangerous toilet
<img width="487" height="386" alt="NVIDIA_Overlay_Mk12xQ539I" src="https://github.com/user-attachments/assets/e710c054-f8df-40e5-ad93-37c15ace6387" />

commons room drained of soul at your behest
<img width="1152" height="755" alt="NVIDIA_Overlay_Ibk4VBiP4f" src="https://github.com/user-attachments/assets/a3f68384-23fd-4c26-a630-728f064f389a" />

cleaned up the med area
<img width="481" height="388" alt="NVIDIA_Overlay_d9JVmsjjmw" src="https://github.com/user-attachments/assets/4a451e5c-174b-438f-af34-fcf751b83b80" />

power output fixed
<img width="463" height="264" alt="NVIDIA_Overlay_57t8hlfzLd" src="https://github.com/user-attachments/assets/35712635-1060-4da1-8f53-ac2267e51e0c" />

redundant firelocks fixed, same with the bright lights
<img width="593" height="398" alt="NVIDIA_Overlay_j9lojqWo1x" src="https://github.com/user-attachments/assets/a36b1ba7-db8d-4392-b4ee-5582ea7629e9" />

Buttons are fixed now

gunnery console swapped
<img width="710" height="551" alt="NVIDIA_Overlay_2apGvBJHgO" src="https://github.com/user-attachments/assets/833749af-0ee0-4051-a6d4-4ef21b1db958" />

ark1368 committed:
Update acera.yml

monolith8319 committed:
chore: automatically update REUSE headers
